packages/cms/form/date-picker/date.tsx

```diff
@@ -0,0 +1,33 @@
+import { EuiDatePicker } from '@tensei/eui/lib/components/date_picker'
+import { EuiFormRow } from '@tensei/eui/lib/components/form'
+import React, { useState } from 'react'
+
+import moment from 'moment'
+import { FormComponentProps } from '@tensei/components'
+
+const FormDatePicker: React.FC<FormComponentProps> = ({
+  field,
+  value,
+  onChange
+}) => {
+  const [startDate, setStartDate] = useState(moment.utc(value))
```

Comment:
Why do we have a shadow state ? 🤔 I think we can just pass the value through without having two states ?